### ISSUE TITLE
Accessibility issues with text-angular toolbar - Not fully keyboard accessible

### ISSUE THREAD

**Issue by katiehockman** [fraywing/textAngular#1559]
There are many accessibility issues with the text-angular toolbar, some of which are **blocking bugs** for assistive technology such as screen readers.

There is currently no way to access the toolbar with the keyboard. One would have to rely on keyboard shortcuts (which wouldn't work for everyone).
A user should be able to navigate to elements on the page, including text and buttons, with a screen reader using the left/right or up/down arrow keys (depending on the screen reader). Currently, if a user navigates to a button on the toolbar with the arrow keys, the button becomes disabled and the keyboard focus is lost.

Expected interaction:
1. User tabs into the text input element.
2. User tabs again, focus moves to the toolbar
3. User navigates to the various buttons in the toolbar using the right and left arrow keys (e.g. a [roving tabindex](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets#Technique_1_Roving_tabindex).
4. User presses the Enter key on the button they want to interact with, triggering the button, and the focus moves back to the part of the input dialog that has changed.
5. User presses tab again to move them back to the toolbar (the focus is now on the button that previously had focus).
6. User presses tab again and focus moves to the next interactable element on the page.
